edith-step/src/ReceiptInput.test.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { shallow, configure, mount } from 'enzyme';
+import ReceiptInput from './ReceiptInput';
+import './setupTests.js'
+
```

Comment:
Is there a way to specifically test if the GroceryList component was  updated when handleSubmit() is called?